src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -457,8 +456,7 @@ private static synchronized void restartLobbyWatcher(final SetupPanelModel setup
 
   public static void resetLobbyHostOldExtensionProperties() {
     for (final String property : getProperties()) {
-      if (GameRunner.LOBBY_HOST.equals(property) || LobbyServer.TRIPLEA_LOBBY_PORT_PROPERTY.equals(property)
-          || GameRunner.LOBBY_GAME_HOSTED_BY.equals(property)) {
+      if (GameRunner.LOBBY_HOST.equals(property) || GameRunner.LOBBY_GAME_HOSTED_BY.equals(property)) {
```

Comment:
Looking at it - that is a mistake, good catch. Fixed in: da4d5fadf8d87e606bb80c5a0900d9e94d4f8deb